winui/Rating/Customization.md

```diff
@@ -231,19 +275,52 @@ Item template selector that uses images to show rated and unrated items.
 
 public class ImageDataTemplateSelector : DataTemplateSelector
 {
-  public DataTemplate SadTemplate { get; set; }
-  public DataTemplate SadUnselectedTemplate { get; set; }
+   public DataTemplate SadTemplate { get; set; }
+   public DataTemplate ExcitedTemplate { get; set; }
+   public DataTemplate HappyTemplate { get; set; }
+   public DataTemplate NeutralTemplate { get; set; }
+   public DataTemplate UnhappyTemplate { get; set; }
+   public DataTemplate SadUnselectedTemplate { get; set; }
+   public DataTemplate ExcitedUnselectedTemplate { get; set; }
+   public DataTemplate HappyUnselectedTemplate { get; set; }
+   public DataTemplate NeutralUnselectedTemplate { get; set; }
+   public DataTemplate UnhappyUnselectedTemplate { get; set; }
+   public DataTemplate SelectedTemplate { get; set; }
+   public DataTemplate ChildTemplate { get; set; }
```

Comment:
@Priyanka-Vijayasankar Remove the unused template.

Reply:
Removed unused template.